src/view/element.js

```diff
@@ -674,6 +674,37 @@ export default class Element extends Node {
 	}
 
 	/**
+	 * Returns identity string based on element's name, styles, classes and other attributes.
+	 * Two elements that {@link #isSimilar are similar} will have same identity string.
+	 * It has the following format:
+	 *
+	 *		"name|classes(class1,class2,class3)|styles(style1=val1,style2=val2)|attributes(attr1=val1,attr2=val2)"
```

Comment:
For instance:
`foo class="baz" style="border-color:white;color:red;" apple="20" banana="10"`
Is much easier for my brain to parse then:
`foo|classes(baz)|styles(border-color=white,color=red)|attributes(apple=20,banana=10)`